tools/agent-ctl/src/client.rs

```diff
@@ -22,6 +22,15 @@ use std::os::unix::net::UnixStream;
 use std::thread::sleep;
 use std::time::Duration;
 use ttrpc;
+use ttrpc::context::Context;
+
+// Hack until the actual Context type supports this.
+fn clone_context(ctx: &Context) -> Context {
```

Comment:
@lifupan, @liubin - can we add a `#[derive(Clone)]` (and maybe `Serialize` and `Deserialize`?) to `ttrpc::context::Context` to remove the need for this at some point?

Reply:
Sorry for the inconvenience, I think that we should add these attributes.